tests/50federation/00prepare.pl

```diff
@@ -11,51 +11,57 @@
 
 my $DIR = dirname( __FILE__ );
 
-push our @EXPORT, qw( INBOUND_SERVER OUTBOUND_CLIENT );
+push our @EXPORT, qw( INBOUND_SERVER OUTBOUND_CLIENT create_federation_server );
+
+sub create_federation_server {
+   my $server = SyTest::Federation::Server->new;
+   $loop->add( $server );
+
+   require IO::Async::SSL;
```

Comment:
There's probably no need to hide this `require` within a function; just move it to toplevel at the start of the file